docs/release-notes/NuGet-5.0-RTM.md

```diff
@@ -0,0 +1,160 @@
+---
+title: NuGet 5.0 RTM Release Notes
+description: Release notes for NuGet 5.0 including known issues, bug fixes, new features, and DCRs.
+author: karann-msft
+ms.author: karann
+ms.date: 04/02/2019
+ms.topic: conceptual
+---
+
+# NuGet 5.0 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version| Available in .NET SDK(s)|
+|:---|:---|:---|
+| [**5.0.0**](https://nuget.org/downloads) | [Visual Studio 2019 version 16.0](https://visualstudio.microsoft.com/downloads/) | 2.1.602<sup>1</sup>, 2.2.202<sup>2</sup> |
+
+<sup>1</sup>Installed with Visual Studio 2019 with .NET Core workload 
+
+<sup>2</sup>Available as an optional install with Visual Studio 2019 with .NET Core workload
+
+## Summary: What's New in 5.0
+
+* Support for restoring [filtered solutions](https://docs.microsoft.com/en-us/visualstudio/ide/filtered-solutions?view=vs-2019) in Visual Studio 2019 - [#5820](https://github.com/NuGet/Home/issues/5820)
+* `BuildTransitive` folder enables packages to transitively contribute targets/props to the host project - [#6091](https://github.com/NuGet/Home/issues/6091)
```

Comment:
Perhaps this would be better:

`BuildTransitive` folder enables packages to contribute targets/props to the host project when the package is used as a transitive reference, instead of a top-level (or direct) `PackageReference`

Reply:
sure... we can match the issue title.